dictionaries/software-terms/src/software-terms.txt

```diff
@@ -907,6 +917,7 @@ KotlinScript
 Kramdown
 kstat
 kurtosis
+kustomization
```

Comment:
This would better fit into a Kubernetes dictionary, especially since it is a misspelling of customization.
Would you be up for making a Kube dictionary? See: [How to Create a Dictionary](https://github.com/streetsidesoftware/cspell-dicts/blob/main/README.md#how-to-create-a-new-dictionary)